linode/manager
# upcoming: [M3-8139, M3-8140, M3-8141] – Add warning notices re: non-encryption when creating Images & enabling Backups

## Description 📝
Add notices to the Imagize dialog, "Enable Backups" dialog, and the Create Image --> "Capture Image" tab advising users that Images and Backups are not encrypted.

## Target release date 🗓️
6/10

## Preview 📷
<details>
<summary>Screenshots</summary>

![Screenshot 2024-05-30 at 11 22 36 AM](https://github.com/linode/manager/assets/114682940/756bfb1b-f4ff-47d6-9f58-9e122ae76762)

![Screenshot 2024-05-30 at 11 30 22 AM](https://github.com/linode/manager/assets/114682940/17017de2-5ecf-44f0-8218-9bcfa25d7707)

![Screenshot 2024-05-30 at 11 31 18 AM](https://github.com/linode/manager/assets/114682940/2c10e17c-c793-4b76-9387-9823485dd3c3)
</details>

## How to test 🧪
### Prerequisites
1. Use an account that doesn't have Managed enabled (since you'll need to check the "Enable Backups" flow)
2. Have the `linode_disk_encryption` tag on your account
3. Point at the alpha/dev environment
4. Have the `Linode Disk Encryption (LDE)` flag in our dev tool toggled on

### Verification steps
Without the tag and/or with the LDE flag in our dev tool toggled off, you should not see any LDE-related things in the UI. Otherwise,

1. Go to the Backups tab on a Linode Details page and click "Enable Backups". Observe: a notice that says "Virtual Machine Backups are not encrypted."
2. Go to the Storage tab on a Linode Details page and select a disk to imagize. Observe in the dialog a notice that says: "Virtual Machine Images are not encrypted."
3. Go to create an Image and on the "Capture Image" tab select a linode in a core region. Observe a notice that says: "Virtual Machine Images are not encrypted." (if you select a linode in an Edge region, you should not see the notice)

The last item is a bit difficult to test at the moment outside of DevEnv, but you can turn the MSW on and first select `metadata-test-image` to see the notice and then switch to the next two (`metadata-test-region` and `Gecko Edge Test`) to see that it doesn't appear. Nonetheless, the `linodeIsInEdgeRegion` logic in `CreateImageTab.tsx` should ensure that the notice is not displayed if the selected linode is in an Edge region.

## As an Author I have considered 🤔
